stellar-auth/pkg/cli/add_user.go

```diff
@@ -62,7 +77,7 @@ func AddUserCmd(databaseURLFlagName string, passwordPrompt PasswordPromptInterfa
 	}
 
 	addUser := &cobra.Command{
-		Use:   "add-user <email> <first name> <last name> [--owner] [--roles] [--password]",
+		Use:   "add-user <email> <first name> <last name> [--owner] [--roles] [--password] [--tenant-id]",
```

Comment:
Isn't the flag `--tenant-id` required? I think we can remove the `[]`.

Reply:
addressed in 09e7ad2bf5ceaff937ffe6cd59ce3796e4e93e11
Thank you!